drivers/nrf_wifi/doc/low_api_common.rst

```diff
@@ -0,0 +1,12 @@
+Common
+######
+
+The common API is used for both normal operation and Radio Test mode.
+
+| Header file: :file:`nrf_wifi/fw_if/umac_if/inc/fmac_structs_common.h`
+| Source file: :file:`nrf_wifi/fw_if/umac_if/src/fmac_api_common.c`
+
+
+.. doxygengroup:: nrf_wifi_api_common
+   :project: nrfxlib
```

Comment:
I guess `nrfxlib` is no longer valid as the project identifier here (similarly in the other two .rst files).

Reply:
Yes, I have updated the project name here, but these files aren't used, once this PR is merged, I will work on doxygen and docs.